### DISCUSSION TITLE
Sharing with an Organization

### DISCUSSION THREAD

**Discussion by rwGP-d3b6nx7Qv5JvqGB** [hassio-addons/addon-bitwarden#203] [Q&A]
I love vaultwarden!  I've noticed the ability to share entries with an organzation.  It it possible to host my own organization so those on my server can share password?  Thanks

**Comment by frenck** [comment]
yes, you can create organizations in the admin panel

**Reply by rwGP-d3b6nx7Qv5JvqGB** [reply]
Thank you!

**Reply by nenonano** [reply]
sorry to rewamp such an old topic but: where am I supposed to create an organization in admin panel?
TIA
